### DISCUSSION TITLE
Not near any campus location for the school, and my last name is wrong?

### DISCUSSION THREAD

**Discussion by GuyverMack** [community/community#120035] [GitHub Education]
### Hi everyone,

Reasons:
`Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.`
and
`You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote . . . .`

My last name is exactly the same on my transcript and my account. They do have my middle name on my transcript, which Github does not ask for, but my last name follows it.

As for the transcript, my school does not put "online" or "distance" on its transcripts. I do not have a student ID because I've always been a distance student. I began as a mail correspondence student while in prison, but I've been released and now I am an online student with a school email address @adams.edu. But my transcripts keep getting rejected.

**Comment by samus-aran** [comment]
Hi @GuyverMack 👋 

I suspect that in regards to reading your name it's your image quality that not high enough for the system to process the text.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Regarding to your distance perhaps reach out to the school and ask them to provide your official documentation stating you are a remote learner. You can include both pieces of evidence in a single photo so that system can capture information on both.
